service.go

```diff
@@ -85,3 +92,44 @@ func handleServiceHelp(dc *downstreamConn, params []string) error {
 	}
 	return nil
 }
+
+func handleServiceCreateNetwork(dc *downstreamConn, params []string) error {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.SetOutput(ioutil.Discard)
+	addr := fs.String("addr", "", "")
+	name := fs.String("name", "", "")
+	username := fs.String("username", "", "")
+	pass := fs.String("pass", "", "")
+	realname := fs.String("realname", "", "")
+	nick := fs.String("nick", "", "")
+
+	if err := fs.Parse(params); err != nil {
+		return err
+	}
+	if *addr == "" {
+		return fmt.Errorf("flag addr is required")
+	}
+
+	if !strings.ContainsRune(*addr, ':') {
```

Comment:
This will be automatically done at connect time, no need to store the hardcoded port in the DB.

(Not storing the hardcoded port allows users to refer to a name-less network more easily, e.g. `chat.freenode.net` instead of `chat.freenode.net:6697`)